apps/public-reference/pages/applications/review/summary.tsx

```diff
@@ -4,21 +4,78 @@ Display a summary of application fields with edit links per section
 */
 import Link from "next/link"
 import Router from "next/router"
-import { Button, FormCard, ProgressNav } from "@bloom-housing/ui-components"
+import { Address } from "@bloom-housing/core"
+import { Button, FormCard, ProgressNav, MultiLineAddress, t } from "@bloom-housing/ui-components"
 import FormsLayout from "../../../layouts/forms"
 import { useForm } from "react-hook-form"
 import { AppSubmissionContext } from "../../../lib/AppSubmissionContext"
 import ApplicationConductor from "../../../lib/ApplicationConductor"
 import { useContext } from "react"
 
+const EditLink = (props: { href: string }) => (
+  <div className="float-right">
+    <Link href={props.href}>
+      <a className="button is-unstyled is-borderless uppercase text-base my-2 text-primary-dark">
+        Edit
+      </a>
+    </Link>
+  </div>
+)
+
+const ReviewItem = (props: { label: string; sublabel?: string; children?: any }) => (
```

Comment:
A better way to type functional components taking children:

```tsx
import type { FunctionComponent } from "react"
const ReviewItem: FunctionComponent<{ label: string; sublabel?: string }> = (props) => (
```

(This will automatically give you `children` with an appropriate type, rather than just using `any`, which will defeat any upstream typechecking. If you want to explicitly type: `children: JSX.Element` is appropriate I think...

Reply:
Hmm, that's pretty verbose and not a pattern we're using anywhere else, but I'll see if I can type `children` more accurately. (needs to handle either string or element(s))

Reply:
Ya, that's the advantage of using the generic type `FunctionComponent` - it takes care of typing `children for you. It's not really much more verbose though:

```
const ReviewItem = (props: { label: string; sublabel?: string; children?: any }) => (
const ReviewItem: FunctionComponent<{ label: string; sublabel?: string }> = (props) => (
```

I _think_ you could use `children: JSX.Element` (or maybe `React.ReactNode`?), but probably worth testing to be sure.